src/collections/_documentation/performance-monitoring/configuration/python.md

```diff
@@ -99,3 +99,51 @@ def process_item(item):
 ```
 
 <!-- ENDWIZARD -->
+
+**Adding Query Information and Parameters to Spans**
```

Comment:
@wmak I added your Python section to the Getting Started instructions. Let me know if there are any inaccuracies. 🌻 Thanks

Reply:
Looks good :+1: